ivy/functional/frontends/jax/numpy/creation.py

```diff
@@ -213,3 +213,23 @@ def csingle(x):
 @to_ivy_arrays_and_back
 def cdouble(x):
     return ivy.astype(x, ivy.complex128)
+
+
+@to_ivy_arrays_and_back
+def compress(condition, a, *, axis=None):
+    condition_arr = ivy.asarray(condition).astype(bool)
```

Comment:
`condition` is already supposed to be a bool array, so I don't think this is necessary.

Reply:
I implemented this functionality because the original function allows the option to pass the `condition` array in the form of 1s and 0s, which can be mapped to bools